### DISCUSSION TITLE
How to fill shape?

### DISCUSSION THREAD

**Discussion by maribox** [flxzt/rnote#1066] [Q&A] [answered]
I want to have a simple rectangle to cover an area of my document, but I can't seem to find a toggle to fill the current shape

**Comment by Doublonmousse** [comment, marked as answer]
![image](https://github.com/flxzt/rnote/assets/115779707/8dbe6257-3b22-4adc-b9f4-294b2cebd4e7)
the second color icon is the fill color

**Comment by maribox** [comment]
Thx <3
